rules/windows/powershell/powershell_accessing_win_api.yml

```diff
@@ -0,0 +1,71 @@
+title: Accessing WinAPI in PowerShell
+id: 03d83090-8cba-44a0-b02f-0b756a050306
+status: experimental
+description: Detecting use WinAPI Functions in PowerShell
+author: Nikita Nazarov
+date: 2020/10/06
+references:
+    - https://speakerdeck.com/heirhabarov/hunting-for-powershell-abuse
+tags:
+    - attack.execution
+    - attack.t1059.001
+    - attack.t1106
+logsource:
+    product: windows
+    service: powershell
+detection:
+    selection:
+        EventID:
+            - 4104
+        Message|contains::
```

Comment:
could you please remove double "`:`"? 
it supposed to be checked on CI/CD test, but they are now disabled for the `oscd` branch, I've asked @thomaspatzke if he can enable it, so we wouldn't have to look for such typoes in the future.

Reply:
Done